### DISCUSSION TITLE
怎样在全屏模式下显示外挂字幕

### DISCUSSION THREAD

**Discussion by Ninosaigo** [Blinue/Magpie#565] [Q&A] [answered]
啃生肉用vnr类工具但是全屏模式下字幕不能显示 怎样能够实现字幕高于画面

**Comment by luojunyuan** [comment]
用powertoys或者deskpins之类的软件把你的外挂字幕窗口置顶就好了

**Reply by Ninosaigo** [reply]
用了很多顶置软件但是都没用，缩放模式只会显示选中窗口，切窗口会自动最小化（但是网易云的桌面歌词可以浮于之上），目前的解决办法只有双屏视野一屏gal一屏字幕

**Reply by luojunyuan** [reply]
你能尝试一下把字幕先拖动到游戏表面上，再缩放试试吗？
这个只能让工具开发者来解决了，让字幕窗口使用一些win32函数在缩放全屏后也保持提前。

**Reply by Ninosaigo** [reply, marked as answer]
感谢，听你这么一说我有了思路，把捕获方式改成desktop dublication就可以了，再次感谢
